### DISCUSSION TITLE
Iteraciones QMP

### DISCUSSION THREAD

**Discussion by santiagobenedetti** [dds-jv/foro#228] [Consultas generales] [answered]
Hola que tal, quería avisar que las iteraciones 3 y 4 de QMP son para este jueves y todavía no esta subida la bitácora.

**Comment by gastonprieto** [comment, marked as answer]
Buenas @sbenedetti21 creo que las el domingo en algún momento del día, pero ya deberían estar, en particular la que hay que entregar es la #4
